core/lib/dal/.sqlx/query-24fa393b7c3a0178806a223ad474e13d46b5ea54a144ec2345a76b865c85f84b.json

```diff
@@ -1,28 +0,0 @@
-{
-  "db_name": "PostgreSQL",
-  "query": "\n            SELECT\n                initiator_address,\n                MAX(nonce)\n            FROM\n                transactions\n            WHERE\n                initiator_address = ANY ($1)\n                AND is_priority = FALSE\n                AND (\n                    miniblock_number IS NOT NULL\n                    OR error IS NULL\n                )\n            GROUP BY\n                initiator_address\n            ",
-  "describe": {
-    "columns": [
-      {
-        "ordinal": 0,
-        "name": "initiator_address",
-        "type_info": "Bytea"
-      },
-      {
-        "ordinal": 1,
-        "name": "max",
-        "type_info": "Int8"
-      }
-    ],
-    "parameters": {
-      "Left": [
-        "ByteaArray"
-      ]
-    },
-    "nullable": [
-      false,
-      null
-    ]
-  },
-  "hash": "24fa393b7c3a0178806a223ad474e13d46b5ea54a144ec2345a76b865c85f84b"
-}
```

Comment:
why is this deleted?

Reply:
It was added here https://github.com/matter-labs/zksync-era/commit/22099cb2f90d917a2479b82c8a4165e68c8da878#diff-9a8298d90368acb2dd2e979f9a121d6106ae200c18ce082252bb4b17e693da5e, probably this query was present in some intermediate version of Danil's PR and he didn't rerun `sqlx prepare` after it was removed